app/src/main/java/fr/groggy/racecontrol/tv/ui/home/HomeItemCardView.kt

```diff
@@ -0,0 +1,32 @@
+package fr.groggy.racecontrol.tv.ui.home
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.leanback.widget.BaseCardView
+import fr.groggy.racecontrol.tv.R
+
+class HomeItemCardView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = R.style.TextCardStyle
+): BaseCardView(
+    context,
+    attrs,
+    defStyle
+) {
+    private val textView: TextView
+
+    init {
+        LayoutInflater.from(getContext())
+            .inflate(R.layout.home_item_card, this)
```

Comment:
This can be replaced by the View `inflate` method...

inflate(context, R.layout.home_item_card, this)